quinn-proto/src/connection.rs

```diff
@@ -1582,13 +1582,13 @@ impl Connection {
         }
 
         // CRYPTO
-        while buf.len() + 16 < max_size {
+        while buf.len() + 17 < max_size {
```

Comment:
I think this really wants to be a constant (probably in `packet`?). Maybe we can use it in other places, too.